drivers/rtc/rtc_ll_stm32.c

```diff
@@ -346,16 +334,24 @@ static int rtc_stm32_set_calibration(const struct device *dev, int32_t calibrati
 	}
 
 	/* wait for recalibration to be ok if a previous recalibration occurred */
-	if (!WAIT_FOR(LL_RTC_IsActiveFlag_RECALP(RTC) == 0, 100000, k_msleep(1))) {
+	if (!WAIT_FOR(LL_RTC_IsActiveFlag_RECALP(RTC) == 0, 100000, 100000, k_msleep(1))) {
```

Comment:
Will this build ?

Reply:
I didn't have CONFIG_RTC_CALIBRATION active, so it was built for me, but it's a mistake; thanks for letting me know. I fixed that.